### PR TITLE
doc(dev-guide): note good first issue

### PR DESCRIPTION
**Pull request by xxchan**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added test labels as necessary. See [details](https://github.com/risingwavelabs/risingwave/blob/main/docs/developer-guide.md#ci-labels-guide).
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

**Review by stdrc — CHANGES_REQUESTED** [review]
description

**Comment by xxchan** [comment]
> description

no need

File: docs/dev/src/contributing.md [hunk 1]
```diff
@@ -11,12 +11,26 @@ To report bugs, create a [GitHub issue](https://github.com/risingwavelabs/rising
...
+
 ## Tests and miscellaneous checks
 
 Before submitting your code changes, ensure you fully test them and perform necessary checks. The testing instructions and necessary checks are detailed in other sections of the book.
 
 ## Submit a PR
 
+### Ask for Review
+
+Related developers may not directly notice your PR. You can find developers who recently edited
```
Comment (xxchan) [inline docs/dev/src/contributing.md:27]:
```suggestion
Related developers may not directly notice your PR. To get your PR merged sooner, you may find reviewers on your own. You can find developers who recently edited
```

File: docs/dev/src/contributing.md [hunk 2]
```diff
@@ -11,12 +11,26 @@ To report bugs, create a [GitHub issue](https://github.com/risingwavelabs/rising
...
+the same file(s) as you did, and `@` them to review your PR. If you are not sure, you may also
+`@` any active developers and ask them to find related reviewers.
+
+If you got no response after a round of review, always feel free to ping reviewers again.
+This can make it much faster to get your PR merged.
+
```
Comment (kwannoel) [inline docs/dev/src/contributing.md:33]: Shall we also refer them to community slack?

**Review by kwannoel — APPROVED** [review]
LGTM, thank you

**Review by stdrc — APPROVED** [review]

**Pull request merged**